src/seed_data/factories/Course.factory.ts

```diff
@@ -0,0 +1,43 @@
+import { define } from 'typeorm-seeding';
+import { Course } from '../../entity/Course';
+import { subjects, getRandomIntInclusive, toUpperCase } from '../utils'
+
+import * as faker from 'faker';
+
+faker.seed(123);
+
+let courseNumbers = [];
+
+function generateCourseNumber() {
+    let num = faker.random.number();
+    if (num < 1000) {
+        num *= 1000;
+    }
+
+    let stringNum = num.toString().substring(0, 4);
```

Comment:
if we're able to that range thing above we wouldn't need to parse it as a string